packages/features/bookings/lib/timePreferences.ts

```diff
@@ -12,6 +12,7 @@ type TimePreferencesStore = {
 };
 
 const timezoneLocalStorageKey = "timeOption.preferredTimeZone";
+const currentTimezone = dayjs.tz.guess() !== "Etc/Unknown" ? dayjs.tz.guess() : "Europe/London";
```

Comment:
`Etc/Unknown` timezone states that timzone information cannot be determined based on user's preferences.
More info about the issue:- https://github.com/iamkun/dayjs/issues/2560

Reply:
Yeah, i will update it

Reply:
done